src/main/java/woohakdong/server/api/controller/admin/auth/dto/AdminJoinRequest.java

```diff
@@ -0,0 +1,8 @@
+package woohakdong.server.api.controller.admin.auth.dto;
+
+public record AdminJoinRequest(
+        String username,
+        String name,
+        String email
+) {
+}
```

Comment:
프론트 분들 안헷갈리시려면, 이전의 회원가입이랑 동일하게 필드명으로 가져가야할 것 같습니다

```suggestion
public record AdminJoinRequest(
        String memberName,
        String name, // <- ?
        String memberEmail
) {
}
```